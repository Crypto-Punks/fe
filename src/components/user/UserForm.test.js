import React from 'react';
import { shallow } from 'enzyme';
import UserForm from './UserForm';
import { Provider } from 'react-redux';
import store from '../../store';

const data = {
  buttonText: 'test',
  redirectText: 'test',
  redirectLink: 'test',
  error: 'test',
  handleSubmit: () => {},
  handleClearError: () => {}
};

describe('UserForm component', () => {
  it('renders UserForm', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <UserForm { ...data } />
      </Provider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});