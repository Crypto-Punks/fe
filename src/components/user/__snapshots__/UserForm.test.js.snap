// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm component renders UserForm 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <UserForm
    buttonText="test"
    error="test"
    handleClearError={[Function]}
    handleSubmit={[Function]}
    redirectLink="test"
    redirectText="test"
  />
</ContextProvider>
`;
